src/cli/options/options_test.go

```diff
@@ -60,3 +60,50 @@ func (suite *OptionsUnitSuite) TestAddExchangeCommands() {
 	err := cmd.Execute()
 	require.NoError(t, err, clues.ToCore(err))
 }
+
+func (suite *OptionsUnitSuite) TestDisableConcurrencyLimiterFlag() {
+	tests := []struct {
+		name     string
+		args     []string
+		assertFn func(*testing.T, bool, string)
+		addFlag  bool
+	}{
+		{
+			name: "--disable-concurrency-limiter not set",
+			args: []string{"test"},
+			assertFn: func(t *testing.T, f bool, s string) {
+				assert.False(t, f, s)
+			},
+			addFlag: false,
+		},
+		{
+			name: "--disable-concurrency-limiter set",
```

Comment:
I think this case is better covered by adding the flag to the `TestAddExchangeCommands` above.  After that, I'm not sure we need a test case for not adding the flag, since the addition of the flag itself is guaranteed, not conditional.

Reply:
Fair point. I wanted to add a test case to assert on default value of the flag, which is false. If it's too extra, I can remove  this test and add to `TestAddExchangeCommands`

Reply:
Update: I think there is value to checking for default flag behavior. So i've kept this as is.

Reply:
Good idea. Let me do this. Although I prefer dedicated unit tests per flag, but I think your suggestion is more reasonable.

Reply:
Oh wait. My understanding was wrong. The flag defaults only get set when the Add*(cmd) functions are called. 
If the Add* funcs are not called, these flags would be initialized to false(or 0). Hence testing for flag == false is moot. 

I'm gonna add to `TestAddExchangeCommands` and call it done. There is no value add testing for defaults.